docs/topics/install/help/index.md

```diff
@@ -5,21 +5,16 @@ We recommend edgectl for [installing Ambassador Edge Stack](../../../tutorials/g
 - [Could not find a Kubernetes cluster](no-cluster)
 - [Email request interrupted](email-request)
 - [Failed to create a Host resource in your cluster](host-resource-creation)
-- [Failed to download AES manifests](aes-manifests)
```

Comment:
No, we can't remove or otherwise repurpose existing pages because they are already out there "in the wild". Rather, we need new pages for the (smaller number of) ways the Helm install can fail in these parts of the installation.

Reply:
Ok, fixed.